src/os/ui/wiz/wizard.js

```diff
@@ -120,7 +120,7 @@ os.ui.wiz.WizardCtrl = function($scope, $element, $timeout, $attrs) {
  */
 os.ui.wiz.WizardCtrl.prototype.destroy_ = function() {
   var step = this['steps'][this['activeIndex']];
-  if (!step.isDeactivated()) {
+  if (!step.isDeactivated() && this.config) {
```

Comment:
The real underlying problem here is that the wizard steps support generics, but the wizard itself is not defined with the necessary generics type info to pass on to the steps. That in itself is outside the scope of this issue. However, checking for null config here in `wizard.js` prevents `IWizardStep` instances defined with `null` types from being used to their full extent.

In other words, instead of doing this here, I'd rather see the check for null config inside each `step.deactivate()` since that will allow some types to support null config if they want to.

Reply:
Ok that makes sense!